tests/sweep_framework/runner.py

```diff
@@ -74,49 +88,47 @@ def execute_suite(test_module, test_vectors, pbar_manager, suite_name):
             result["status"] = TestStatus.NOT_RUN
             result["exception"] = "INVALID VECTOR: " + test_vector["invalid_reason"]
             result["e2e_perf"] = None
-            result["timestamp"] = datetime.datetime.now().strftime("%Y-%m-%d_%H-%M-%S")
-            results.append(result)
-            suite_pbar.update()
-            continue
         else:
             test_vector.pop("invalid_reason")
             test_vector.pop("status")
             test_vector.pop("validity")
-        if p is None:
-            p = Process(target=run, args=(test_module, input_queue, output_queue))
-            p.start()
-        try:
-            if MEASURE_PERF:
-                # Run one time before capturing result to deal with compile-time slowdown of perf measurement
+            if p is None:
```

Comment:
Some of these changes seem unrelated to the PR?

Reply:
It's just to avoid duplicating the "host" and "user" assignments if there is an invalid vector, just moves the running to the else block of the invalid check and assigns the host and user after regardless of if the vector ran.